pkg/network/protocols/postgres/model_linux_test.go

```diff
@@ -24,6 +24,11 @@ func TestExtractTableFunction(t *testing.T) {
 			query:      `DROP TABLE IF EXISTS test1`,
 			tablesName: "test1",
 		},
+		{
+			name:       "single table name",
+			query:      `drop table if exists test1`,
+			tablesName: "test1",
+		},
```

Comment:
either change the case to have mixed letters (`DrOp taBLE If EXIsts ....`) or add a case of a mixed letters